velodyne_pointcloud/include/velodyne_pointcloud/rawdata.hpp

```diff
@@ -121,8 +139,31 @@ struct raw_packet
 class RawData final
 {
 public:
+  // RawData();
   RawData(const std::string & calibration_file, const std::string & model);
 
+  ~RawData()
+  {
+  }
```

Comment:
Please remove this empty destructor. It disables copy/move constructors and isn't being used.